pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -80,6 +80,14 @@ import (
 const (
 	PCI_RESOURCE_PREFIX  = "PCI_RESOURCE"
 	MDEV_RESOURCE_PREFIX = "MDEV_PCI_RESOURCE"
+
+	baseEphemeralDiskDir = "/var/run/kubevirt-ephemeral-disks/"
+)
+
+var (
+	getEphemeralDiskBaseDir = func() string {
```

Comment:
Maybe we can even align, so we don't declare the consts twice: https://github.com/kubevirt/kubevirt/pull/6807/files#diff-7a3f9e0461f554b07ac195375eb0f8db7d46579071795a65c459c721b1a8204aR1148

WDYT?